Packet++/header/ProtocolType.h

```diff
@@ -327,6 +327,11 @@ namespace pcpp
 	 */
 	const ProtocolType SLL2 = 0x8000000000000;
 
+	/**
+	 * S7COMM protocol
+	 */
+	const ProtocolType S7COMM = 0x9000000000000;
```

Comment:
This is a bit array represented by a hex value. Each protocol show have only 1 bit set. So the next value after `0x8000000000000` is `0x10000000000000`